docs/guides/c-extension-for-python.mdx

```diff
@@ -15,7 +15,7 @@ This requires additional steps to the standalone C usage; for more details, see
 </Admonition>
 
 <Admonition type="note">
-  These instructions have only been tested on UNIX-like systems. Windows instructions are in progress._
+  These instructions have only been tested on UNIX-like systems. Windows instructions are in progress.
```

Comment:
Are windows instructions still in progress?

Reply:
@beckykd and @christopherporter1 are my two go-tos for Windows systems :D